api/src/test/unit/java/uk/gov/dvsa/mot/trade/read/core/TradeReadServiceTest.java

```diff
@@ -563,6 +568,127 @@ public void getLatestMotTestByMotTestNumberWithSameRegistrationAndVin_VehicleWit
         assertEquals("Secondary colour is incorrect", secondaryColour, apiVehicle.getSecondaryColour());
     }
 
+    @Test
+    public void getLatestMotTestByDvlaVehicleId_ReturnsVehicle_FullyPopulated_WhenMotTestExists() {
+
+        final Integer dvlaVehicleId = 123;
```

Comment:
I guess so, although I don't think it's a bad thing keeping tests as contained as possible.